localisation/00_zuwc_united_localisation.csv

```diff
@@ -2894,6 +2893,104 @@ GetSocietyWe_society;[This.Society.GetName];;;;;;;;;;;;;x
 GetSocietyWe_we;we;;;;;;;;;;;;;x
 nick_the_calculator;the Calculator;;;;;;;;;;;;;x
 nick_the_shatterer;the Shatterer;;;;;;;;;;;;;x
+BM_ASCENSION_OF_THE_LICH_KING;Ascension of the Lich King;;Aufstieg des Lichk�nigs;;;;;;;;;;;x
+BM_ASCENSION_OF_THE_LICH_KING_DESC;Azeroth is still reeling from the devastation caused by the horrors of the Third War. The kingdoms of northern Azeroth lie in ruins, consumed by the relentless march of the undead Scourge. The forests of Kalimdor bear the horrible taint of the demonic Burning Legion, despite their spectacular defeat at the summit of Mount Hyjal. Nevertheless, new developments occur as the orcs finally begin to settle the red sands of Durotar and the fractured Alliance deals with threats on every front, including domestic ones. The undead Scourge is also in dire straits, as the Illidari return to Azeroth in order to finish the job they started in Dalaran, while the undead forces in Lordaeron are on the brink of civil war...;;Azeroth leidet noch immer unter den Verw�stungen, die die Schrecken des dritten Krieges verursacht haben. Die K�nigreiche im Norden Azeroths liegen in Tr�mmern, verzehrt vom unerbittlichen Marsch der untoten Gei�el. Die W�lder von Kalimdor leiden unter dem furchtbaren Einfluss der d�monischen brennenden Legion, trotz deren spektakul�rer Niederlage auf dem Gipfel des Berges Hyjal. Gleichwohl kommt es zu neuen Entwicklungen, da die Orks endlich beginnen, die roten Sande von Durotar zu besiedeln, und die zerbrochene Allianz mit Bedrohungen an allen Fronten, auch im Inland, fertig werden muss. Auch die Gei�el der Untoten befindet sich in einer ernsten Notlage, da die Illidari nach Azeroth zur�ckkehren, um ihre in Dalaran begonnene Arbeit zu beenden, w�hrend die untoten Streitkr�fte in Lordaeron am Rande eines B�rgerkriegs stehen...;;;;;;;;;;;x
```

Comment:
Why do you want all mentions of races capitalised? Blizzard doesn't do it eiter...

Reply:
I would assume ALL the Orcs in durotaur are doing the settling, it makes sense to me since when I see fantasy races/species referred to as a whole, capitalizing makes more sense,
But if you want to follow blizzard's style, then just reject the changes and state that's why you did it.

Reply:
using blizzard/non-fictional writing style for races/racial adjectives